test/integration/scheduler/framework_test.go

```diff
@@ -216,3 +245,44 @@ func TestPrebindPlugin(t *testing.T) {
 		cleanupPods(cs, t, []*v1.Pod{pod})
 	}
 }
+
+// TestPostbindPlugin tests invocation of postbind plugins.
+func TestPostbindPlugin(t *testing.T) {
```

Comment:
As I said above, a post-bind plugin does not return a value. A good test should check that a post-bind plugin is called one time when a pod is successfully bound and is not called when a pod fails to bind.
In order to write such test, one option is to register a pre-bind and a post-bind plugin. In order to simulate binding failure, you can make the pre-bind plugin return failure. Then you check that the post-bind plugin is not called.
In a different case, you check that the post-bind plugin is called once when pre-bind passes.

Reply:
DONE